hardware_layer/hardware_arduino/firmware/fourtran_test.cpp

```diff
@@ -68,6 +68,7 @@ void motor1_PWM_callback(const std_msgs::Int32& msg)
 {
   motor1.spin(msg.data);
   nh.loginfo("Motor1 data received");
+  // nh.logininfo("Motor1 data: %d",msg.data);
```

Comment:
Not required. All the loginfo statements ` nh.logininfo("Motor# data: %d",msg.data);` should be removed since logingfo does not stupport `ROS_INFO` kind of style.